gwpy/cli/cliproduct.py

```diff
@@ -41,7 +41,7 @@
 
 from dateutil.parser import parser
 
-from ..timeseries import TimeSeries
+from gwpy.timeseries import TimeSeries
```

Comment:
@areeda, what's the reason for this change?

Reply:
I think it's a tad more obvious plusI can copy/pasteinto ipython
No big deal I can reverse it if you likr

Reply:
yes, please revert. since this module is part of `gwpy`, it should use relative imports for relative objects as much as possible.